galaxy.yml

```diff
@@ -0,0 +1,20 @@
+namespace: community
```

Comment:
Add min ansible version?

Reply:
@gundalow - That would be under `meta/runtime.yml` right? (like https://github.com/ansible-collections/community.kubernetes/blob/master/meta/runtime.yml#L2)

Reply:
ah, thanks. I was getting confused why I couldn't find it.

@geerlingguy @felixfontein Do you think this would be good to include in the default template?

Reply:
Why not. I think the default value should be 2.9.10.